app/components/intermission.jsx

```diff
@@ -0,0 +1,202 @@
+// https://github.com/EnCiv/civil-pursuit/issues/137
+import React, { useState, useEffect } from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+import { PrimaryButton, SecondaryButton } from './button'
+import Intermission_Icon from '../svgr/intermission-icon'
+const Intermission = props => {
+  const {
+    className = '',
+    user = {},
+    round = 1, // the round that the user has just completed
+    ...otherProps
+  } = props
+  const classes = useStylesFromThemeFunction(props)
+
+  const [validationError, setValidationError] = useState(null)
+  const [successMessage, setSuccessMessage] = useState(null)
+  const [useremail, setUseremail] = useState(null)
+
+  const validateEmail = email => {
+    var re =
+      /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+    return re.test(String(email).toLowerCase())
+  }
+  const handleEmail = () => {
+    let email = user.email
+    console.log(email)
+    console.log(user.tempid)
+    if (!validateEmail(email)) {
+      setValidationError('email address not valid')
+      setSuccessMessage(null)
+      setUseremail(false)
+    } else {
+      setValidationError(null)
+      setUseremail(true)
+    }
+
+    console.log(validationError)
```

Comment:
We are going to have to let the user know about the validation error and the success message.  See the sign-on component for how it shows messages like this.